components/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/RequestExecutor.java

```diff
@@ -87,10 +120,19 @@ public void execute(WorkflowRequest workFlowRequest) throws WorkflowException {
         validateExecutionParams();
         OMElement requestBody = WorkflowRequestBuilder.buildXMLRequest(workFlowRequest, this.parameterList);
         try {
-            callService(requestBody);
+            String templateId = getTemplateIdByWorkflowId(parameterList.get(0).getWorkflowId());
```

Comment:
```suggestion
            String templateId = getTemplateIdByWorkflowId(this.parameterList.get(0).getWorkflowId());
```

Reply:
addressed the comment ref:https://github.com/wso2-extensions/identity-workflow-impl-bps/pull/70/commits/641aaf120ea3a609b753a77ae028dae66a9ffa98